src/component/molecules/navbarMenuLeft/navbarMenuLeftCustomize.tsx

```diff
@@ -0,0 +1,7 @@
+import { Box } from "@mui/material";
+import styled from "styled-components";
+
+export const BoxCustomize = styled(Box)`
```

Comment:
It is not looking good. I mean, look. You name this in a very general way but keep in `navbar` things. 
Betters name or move it to a proper dir for general things.